native/.watchmanconfig

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
empty config? what happens if we delete it ?

Reply:
Let's find out!

Reply:
It seems to be fine with it deleted!